app/components/layout.js

```diff
@@ -37,12 +37,18 @@ class Layout extends React.Component {
   }
 
   render() {
+    let toolbelt = null;
+    if (this.props.status === 'CONNECTED') {
```

Comment:
This should be a function, I think I was a bit ambiguous. My idea is, if this is put inside a function, the will be no need to set the `toolbelt` value to `null`

```javascript
const renderToolbelt = (props, handshake, cpuProfiling) => {
  if (props.status === 'CONNECTED') {
    return <Toolbelt {...props} handshake={handshake} handleCpuSampling={cpuProfiling} />;
  } else {
    return null;
}
```

But I'm going to merge this now.